src/tactic/ui/table/task_element_wdg.py

```diff
@@ -1817,13 +1830,64 @@ def get_task_wdg(my, tasks, parent_key, pipeline_code, process, last_one):
             color = Task.get_default_color(status)
 
 
+
+            complete = my.get_complete(sobject, related_type, related_process, related_scope, related_pipeline_code=related_pipeline_code)
+
+
+            num_complete = 0
+            complete_search_keys = []
+            for key, value in complete.items():
+                if value:
+                    num_complete += 1
+
+                search_key, process, key = key.split("|")
+                complete_search_keys.append(search_key)
+
+
+            count = num_complete
+            total = len(complete)
+
+
+
+            from spt.ui.widgets import RadialProgressWdg
+            progress_wdg = RadialProgressWdg(
+                total=total,
+                count=count,
+                color=color
+            )
+
+
             div.add_behavior( {
                 'type': 'click_up',
                 'code': code,
                 'scope': related_scope,
                 'search_type': search_type,
                 'related_type': related_type,
+                'search_keys': complete_search_keys,
                 'cbjs_action': '''
+
+                var class_name = 'tactic.ui.table.RelatedTaskWdg';
+                var kwargs = {
+                    related_type: bvr.related_type,
+                    search_keys: bvr.search_keys,
+                }
+
+                var server = TacticServerStub.get();
+                var sobject = server.get_by_code(bvr.search_type, bvr.code);
+                spt.tab.set_main_body_tab();
+                var name = sobject.name;
+                if (!name) {
+                    name = sobject.code;
+                }
+                name = "Related: " + name
+                var title = name;
+                spt.tab.add_new(name, title, class_name, kwargs);
+
+
+                return;
+ 
+
+
                 var class_name = 'tactic.ui.panel.ViewPanelWdg';
                 if (bvr.scope == "global") {
```

Comment:
Should we comment out these whole expression if else block if you are are using it any more? you are using complete_search_keys instead